fuzz/tls_init/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+project(fuzz_init CXX)
+
+set(CMAKE_C_FLAGS ${CMAKE_C_FLAGS})
+set(CMAKE_CXX_COMPILER "/usr/bin/clang++")
+set(CMAKE_CXX_FLAGS "-fsanitize=address,fuzzer -g -fPIE ${CMAKE_CXX_FLAGS}")
+
+set(INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/../../src/include
+                    ${CMAKE_CURRENT_SOURCE_DIR}/../../src/include/edl
+                    ${CMAKE_CURRENT_SOURCE_DIR}/../../src/include/rats-tls
+                    ${RATS_TLS_INSTALL_INCLUDE_PATH}
+                    ${RATS_TLS_INSTALL_INCLUDE_PATH}/edl
+                    )
+set(LIBRARY_DIRS ${RATS_TLS_INSTALL_LIB_PATH})
+
+include_directories(${INCLUDE_DIRS})
+link_directories(${LIBRARY_DIRS})
+
+# Set source file
+set(SOURCES fuzz_init.cc)
+
+# Generate bin file
+add_executable(${PROJECT_NAME} ${SOURCES})
+target_link_libraries(${PROJECT_NAME} rats_tls)
+
+install(TARGETS ${PROJECT_NAME}
+	DESTINATION /usr/share/rats-tls/fuzz)
```

Comment:
please use ${RATS_TLS_INSTALL_FUZZ_PATH} instead of /usr/share/rats-tls/fuzz to facilitate subsequent maintenance